### DISCUSSION TITLE
微博V7（新版）字体阴影尺寸（字重）不一致

### DISCUSSION THREAD

**Discussion by xiaoxutem** [F9y4ng/GreasyFork-Scripts#153] [问答专区(Question / Answer)]
![微信截图_20221219114650](https://user-images.githubusercontent.com/103314132/208371563-dc907fc1-a210-4ac6-a3e0-221532dce863.png)
![微信截图_20221219113918](https://user-images.githubusercontent.com/103314132/208371575-295fe203-a056-470a-b69b-c56fc5339035.png)

**Comment by F9y4ng** [comment]
这个问题好像在早期版本时有个用户提到过。 #49 

造成这样的视觉差异问题应该是浏览器对不同大小(font-size)字体的渲染呈现出不同的效果。一般来说，字体越大效果越好，字体越小效果越差（通常和具体的字体渲染参数有关，大都是字体越小越模糊，甚至变得一坨黑的感觉）。这与浏览器或系统缩放有点类似，缩放的越大，字体越清晰。

新版的字体渲染参数中，考虑到了这个问题，其修正效果由于显示器、浏览器的不同而因人而异，但可尝试一下：

## 建议操作
1. 尝试更换其他中文字体。如果必须使用微软雅黑，在低分辨率下，尽量不要使用字体描边，因为此种情况下浏览器渲染有渲染偏移的bug。
2. 适当降低字体阴影尺寸。
3. 调整字体阴影颜色的alpha值，增加透明度。（推荐使用深色高透明度来呈现同样效果的色彩）

_如以上操作后仍未达到你个人的审美要求，那说明本脚步暂时无法解决你遇到的问题。_
